receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -8,12 +8,15 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"strings"
 	"time"
 
+	promConfig "github.com/prometheus/prometheus/config"
```

Comment:
nit: it is unusual to alias imports as camelcase in go.  Prefer `promconfig` to `promConfig`.